parts/linux/cloud-init/artifacts/ipv6_nftables.service

```diff
@@ -1,5 +1,6 @@
 [Unit]
 Description=Configure nftables rules for handling Azure SLB IPv6 health probe packets
+Before=kubelet.service
```

Comment:
is there any scenario in which `ipv6_nftables.sh` could fail and prevent kubelet from starting? From a quick look at the script, it seems safe enough (just curl to IMDS, jq, then running `nft` to install the rules)